core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -282,14 +294,23 @@ private SELF self() {
 }
 
 class DockerCompose extends GenericContainer<DockerCompose> {
-    public DockerCompose(File composeFile, String identifier) {
+    public DockerCompose(List<File> composeFiles, String identifier) {
 
         super("docker/compose:1.8.0");
+        checkNotNull(composeFiles);
+        checkArgument(!composeFiles.isEmpty(), "No docker compose file have been provided");
+
         addEnv("COMPOSE_PROJECT_NAME", identifier);
+
         // Map the docker compose file into the container
-        String pwd = composeFile.getAbsoluteFile().getParentFile().getAbsolutePath();
-        addEnv("COMPOSE_FILE", pwd + "/" + composeFile.getAbsoluteFile().getName());
+        File dockerComposeMainFile = composeFiles.get(0);
+        final String pwd = dockerComposeMainFile.getAbsoluteFile().getParentFile().getAbsolutePath();
+        List<String> absoluteDockerComposeFiles = composeFiles.stream().map(file -> pwd + "/" + file.getAbsoluteFile().getName()).collect(Collectors.toList());
```

Comment:
The benefit of "/" is that it works both on windows and unix like systems. This why used it instead of File.separator